acceptance/README.md

```diff
@@ -3,28 +3,41 @@ This folder contains acceptance tests that are required for Chef client
 release readiness.
 
 ## Getting started
-The tests use the _chef-acceptance_ gem as the high level framework.
+The tests use the [chef-acceptance](https://github.com/chef/chef-acceptance) gem as the high level framework.
 All the gems needed to run these tests can be installed with Bundler.
 
 ### Important Note!
-Before running chef-acceptance, you *MUST* do the following on your current session:
+If you are using Chef Development Kit's ruby environment make sure you  do the following on your current session:
 
 ```
 export APPBUNDLER_ALLOW_RVM=true
 ```
 
 ### Setting up and running a test suite
-To get started, do a bundle install from the acceptance directory:
-```shell
-chef/acceptance$ bundle install
-```
 
-To get some basic info and ensure chef-acceptance can be run, do:
 ```shell
-chef/acceptance$ bundle exec chef-acceptance info
+$ cd acceptance
```

Comment:
Because this README is already in the acceptance folder, I actually think we don't need this line.  Instead it could be `# You should be in the 'chef/acceptance' folder to run the following`